runtime/syntax/scons.yaml

```diff
@@ -0,0 +1,7 @@
+filetype: scons
+
+detect:
+    filename: "(\\SConscript$|\\SConstruct$)"
+
+rules:
+    - include: "python"
```

Comment:
Why not add it to the python syntax file instead? (We were already discussing this in #3184.)

Reply:
> Why not add it to the python syntax file instead? (We were already discussing this in #3184.)

In general it is orphaned PR which was created even before that discussion